containers/datalab/content/run.sh

```diff
@@ -54,6 +54,21 @@ check_tmp_directory() {
     echo "The /tmp directory is writable"
 }
 
+# Reinstall the parts of pydatalab from /content/pydatalab, which is where it gets live-mounted.
+function reinstall_pydatalab() {
+  PYDATALAB=/content/pydatalab
+  echo "Reinstalling pydatalab from ${PYDATALAB}"
+  pip uninstall -y datalab
```

Comment:
OK, I took out the "pip uninstall", it still logs that it is uninstalling before it installs, so that looks like it is working.